src/Logout Navigator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
why are there 2 logoutNavigator files?

Reply:
removed extra file and also refactored PUT route and moved LOGOUT to health stats